pkg/operator/controller/ingress/status_test.go

```diff
@@ -341,6 +360,8 @@ func TestComputeIngressDegradedCondition(t *testing.T) {
 			},
 			expectIngressDegradedStatus: operatorv1.ConditionFalse,
 			expectRequeue:               true,
+			// Minimum grace period of combined conditions (DNSReadyIngressConditionType) is 30 seconds
```

Comment:
```suggestion
			// Minimum grace period of combined conditions (LoadBalancerReadyIngressConditionType) is 30 seconds
```

Reply:
DNSReadyIngressConditionType grace period is the minimum at 30 seconds.  LoadBalancerManagedIngressConditionType grace period is 90 seconds.

Reply:
Sorry, I must have been confused.  You're right; this test case looks fine.